### DISCUSSION TITLE
Cant intall Xubuntu on m1 Mac UTM

### DISCUSSION THREAD

**Discussion by mkztx** [utmapp/UTM#4238] [UTM Help (macOS)]

**Comment by Nanoc-ice** [comment]
I have a similar problem with any VM trying to install on the machine.  It errors out right away with an error and a bunch of chinese words. i exported the qemu debug and command text which looks like the outstanding thing i see is the argument for spice.  Does the qemu on the m1 require spice to be in qemu now on this machine?

**Comment by Nanoc-ice** [comment]
this is closed for me.  had to get UTM out of quarantined state with:
sudo xattr -d com.apple.quarantine -r /Applications/UTM.app

**Comment by omerien** [comment]
MarcinLitwin don't expect support if you don't give any information.
